package.json

```diff
@@ -1,38 +1,27 @@
 {
 	"name": "presentationtools-cuetimer",
-	"version": "1.1.4",
-	"api_version": "1.0.0",
+	"version": "2.0.0",
 	"support": {
```

Comment:
this isn't needed or used. You can safely remove it.

Reply:
Done, 
BTW, I just didn't remove it because it wasn't mentioned in`package.json cleanup` section [here](https://github.com/bitfocus/companion-module-base/wiki/Upgrading-a-module-built-for-Companion-2.x#3-packagejson-cleanup)

Reply:
@Engma90 there was some minor confusion.  The `version` field is required in package.json.  The [manifest.json](https://github.com/bitfocus/companion-module-base/wiki/manifest.json) page explains this.  Can you please add it back in.

Reply:
No worries, it is added now